yocs_joyop/src/joyop.cpp

```diff
@@ -41,6 +44,9 @@ class JoyOp
   bool enable_pressed_, disable_pressed_, deadman_pressed_, brake_pressed_, boost_active_, zero_twist_published_;
   int wait_for_connection_; /**< Time to wait for enable/disable topics in seconds (-1 to not wait). **/
   ros::Timer timer_;
+  
+  dynamic_reconfigure::Server<gopher_core_msgs::BagJoystickConfig> *             dynamic_reconfigure_server;
+  dynamic_reconfigure::Server<gopher_core_msgs::BagJoystickConfig>::CallbackType dynamic_reconfigure_callback;
```

Comment:
Does this actually need to be a class variable? It never goes beyond the scope of the constructor.